src/TokenClient.test.ts

```diff
@@ -87,9 +87,9 @@ describe("TokenClient", () => {
             expect(generateBasicAuthSpy).toHaveBeenCalledWith("client_id", "client_secret");
             expect(getTokenEndpointMock).toBeCalledWith(false);
             expect(postFormMock).toBeCalledWith(
-                "http://sts/token_endpoint",
-                expect.any(URLSearchParams),
-                expect.stringContaining(""),
+                expect.objectContaining({
+                    url: "http://sts/token_endpoint",
+                }),
```

Comment:
This test is a less strict than it was before, the previous revision had checks for URLSearchParams body and the basic auth string. There's several other places within this file of this occurring.